Update installation of yarn on ubuntu, remove legacy, update doc orders (#66)
taichi-dev/docs.taichi.graphics: README.md

yarn --cwd=website install
```

### Trouble shooting

If you are using `ubuntu`, you might get errors as below:
```
Usage: yarn [options]
yarn: error: no such option: --cwd
```

> That is because yarn is too old. You could install new version yarn with `npm`:

Reviewer: ```suggestion
which indicates your  `yarn` is too old. You could install new version yarn with `npm`:
```

Author: > LGTM! It seems that this PR has a lot of overlapping commits with #64 ? Not sure if you want to close the other or rebase after merging the other?

I closed PR #64.
